mon-pix/tests/integration/components/challenge-embed-simulator-test.js

```diff
@@ -54,76 +53,61 @@ describe('Integration | Component | Challenge Embed Simulator', function() {
 
     it('should have text "Réinitialiser"', async function() {
       // when
-      await render(hbs`{{challenge-embed-simulator}}`);
+      await render(hbs`<ChallengeEmbedSimulator />`);
 
       // then
       expect(find('.embed__reboot').textContent.trim()).to.equal('Réinitialiser');
     });
-
-    it('should reload simulator when user clicked on button reload', async function() {
-      // given
-      const stubRebootSimulator = sinon.stub();
-      this.set('stubRebootSimulator', stubRebootSimulator);
-      await render(hbs`{{challenge-embed-simulator _rebootSimulator=stubRebootSimulator}}`);
-
-      // when
-      await click('.embed-reboot__content');
-
-      // then
-      sinon.assert.calledOnce(stubRebootSimulator);
-    });
   });
 
   describe('Blur effect on simulator panel', function() {
 
     it('should be active when component is first rendered', async function() {
       // when
-      await render(hbs`{{challenge-embed-simulator}}`);
+      await render(hbs`<ChallengeEmbedSimulator />`);
 
       // then
-      expect(findAll('.embed__simulator')[0].classList.contains('blurred')).to.be.true;
+      expect(find('.embed__simulator').classList.contains('blurred')).to.be.true;
     });
 
     it('should be removed when simulator was launched', async function() {
       // given
-      await render(hbs`{{challenge-embed-simulator}}`);
+      await render(hbs`<ChallengeEmbedSimulator />`);
 
       // when
       await click('.embed__launch-simulator-button');
 
       // then
-      expect(findAll('.embed__simulator')[0].classList.contains('blurred')).to.be.false;
+      expect(find('.embed__simulator').classList.contains('blurred')).to.be.false;
     });
   });
 
   describe('Embed simulator', function() {
 
-    const embedDocument = {
-      url: 'http://embed-simulator.url',
-      title: 'Embed simulator',
-      height: 200
-    };
-
     beforeEach(async function() {
       // given
-      this.set('embedDocument', embedDocument);
+      this.set('embedDocument', {
+        url: 'http://embed-simulator.url',
+        title: 'Embed simulator',
+        height: 200
+      });
```

Comment:
Peut-être que les `this` pendant les tests sont aussi des instances de classes natives ? Soyons fous !
```suggestion
      this.embedDocument =  {
        url: 'http://embed-simulator.url',
        title: 'Embed simulator',
        height: 200
      });
```

Reply:
Maleureusement, non 😞

Reply:
triste